packages/prerender/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@redwoodjs/prerender",
+  "description": "Redwood prerenter util",
+  "version": "0.25.1",
+  "license": "MIT",
+  "files": [
+    "dist",
+    "dist/index.js",
+    "dist/browser.js"
+  ],
+  "main": "dist/index.js",
+  "peerDependencies": {
+    "react": "*",
+    "react-dom": "*"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.12.10",
+    "@babel/node": "^7.12.10",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/preset-react": "^7.12.10",
+    "@babel/preset-typescript": "^7.12.7",
+    "@babel/register": "^7.12.10",
+    "@redwoodjs/auth": "^0.25.1",
+    "@redwoodjs/internal": "^0.25.1",
+    "@redwoodjs/router": "^0.25.1",
+    "@redwoodjs/web": "^0.25.1",
+    "babel-plugin-ignore-html-and-css-imports": "^0.1.0",
+    "babel-plugin-inline-react-svg": "^1.1.2",
+    "node-fetch": "^2.6.1",
+    "nodemon": "^2.0.7"
+  },
```

Comment:
I'm assuming that a bunch of `devDeps` are required after the package is published. In which case they should be dependencies, since devDeps are not installed by default for user's of a package.

Reply:
Yeah wasn't sure about this one. One of the pain points of lerna/workspaces, because 90% of these dependenceis are already in redwood. 

I'll move this into dependencies and try and match up the versions so we don't duplicate stuff.

Reply:
Deps are super confusing at times, especially when they're satisfied by some other transient package. It's a blessing and a curse, but one that does bite at some point.

I've been using resolutions in the workspace to pin dependencies to the ones that I want, but maybe we can add a package that includes these for us.